scripts/upgrade_admin_server.sh

```diff
@@ -125,6 +139,8 @@ ListenStream=[::]:4369
 FreeBind=true
 EOF
 
+    systemctl start rabbitmq-server
```

Comment:
it seems to be too late for starting rabbitmq. dist-upgrade above includes crowbar packages which try to upload cookbooks and require rabbit to be running.

Reply:
We know ::)
This [patch](https://github.com/crowbar/crowbar/blob/1c9008c6d1704361d6d7f5b75d2e061894b805b9/scripts/upgrade_admin_server.sh#L134) does not work correctly.
I think it is missing a `systemctl daemon-reload`

Reply:
isn't daemon-reload done inside the package scripts? do you know which package changes some unit which doesn't get reloaded?

Reply:
@skazi0 my reason for placing it here was that it is after that `/etc/systemd/system/epmd.socket.d/ports.conf` creation

Reply:
I think we should move both parts above zypper dup. I'm testing it right now. Maybe we need another daemon-reload after ports.conf are created.

Reply:
@jsuchome confirmed: we need another `systemctl daemon-reload` after creating the ports.conf file. So the changes needed for this PR are:
1. move ports.conf creation above zypper dup
2. add `systemctl daemon-reload` after ports.conf are created
3. move `systemctl start rabbitmq-server` above zypper dup but below reload from point 2